.travis.yml

```diff
@@ -65,6 +65,7 @@ script:
   - golint -set_exit_status $(go list ./... | grep -v /vendor/)
   - flake8
   - go test -coverpkg=all -v -coverprofile=coverage.txt -covermode=count gopkg.in/src-d/hercules.v9/... && sed -i '/cmd\/hercules\|core.go/d' coverage.txt
+  - go test -race gopkg.in/src-d/hercules.v9/... # run race test separately to preserve -covermode=count
```

Comment:
note on this: it appears `-race` must be run with `-covermode=atomic`:

```
❯ go test -race -coverpkg=all -v -coverprofile=coverage.txt -covermode=count gopkg.in/src-d/hercules.v9/... && sed -i '/cmd\/hercules\|core.go/d' coverage.txt
-covermode must be "atomic", not "count", when -race is enabled
```

Reply:
I am fine with `-covermode=atomic` as long as it does not break the profile stats, so yeah, let's swap.